packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -796,11 +798,15 @@ export const forkExternalSandbox = async (
     } else {
       state.editor.sandboxes[forkedSandbox.id] = forkedSandbox;
 
-      effects.router.updateSandboxUrl(forkedSandbox, {
-        openInNewWindow,
-        hasBetaEditorExperiment,
-      });
+      if (!preventNavigation) {
```

Comment:
nitpick:  I would reverse this into a param called `redirectAfterFork` that is passed only when it's true and is false by default

Reply:
I'll take a look. I was initially doing `preventNavigation` because we wouldn't have to change all other call-sites to this, but maybe this function is called from a single place.